clang/lib/Driver/ToolChains/ROCm.h

```diff
@@ -272,6 +277,13 @@ class RocmInstallationDetector {
     return Loc->second;
   }
 
+  void init(bool DetectHIPRuntime = true, bool DetectDeviceLib = false) {
```

Comment:
I don't understand why we need this, isn't it fine just to let it fail? Let the detection use the triple and give up if it can't find it with the windows paths.

Reply:
Yes. We are basically just trying to avoid adding "linux like paths" on Windows.  It's cosmetic.